protocol/0028-GOVE-governance.md

```diff
@@ -774,3 +795,11 @@ The voting to approve the batch happens, the batch passes, the value of the prop
 - A voter's equity-like share does not give them any additional voting weight when voting on a market community tags update proposal. (<a name="0028-GOVE-174" href="#0028-GOVE-174">0028-GOVE-174</a>)
 - A proposal to add community tags with any community tags longer than `governance.proposal.market.maxCommunityTagLength` is rejected as invalid (<a name="0028-GOVE-175" href="#0028-GOVE-175">0028-GOVE-175</a>)
 - A proposal to remove community tags with any community tags longer than `governance.proposal.market.maxCommunityTagLength` is rejected as invalid (<a name="0028-GOVE-176" href="#0028-GOVE-176">0028-GOVE-176</a>)
+
+#### Cancelling Proposals
+
+- Given a proposal with a closing time further than `governance.proposal.cancellationThreshold` from the current time, if the party which created the proposal votes against it, the proposal will be instantaneously rejected (<a name="0028-GOVE-188" href="#0028-GOVE-188">0028-GOVE-188</a>)
+- Given a proposal with a closing time less than `governance.proposal.cancellationThreshold` from the current time, if the party which created the proposal votes against it, the proposal will not be instantaneously rejected. At closing time, if the proposal meets the criteria to be accepted it will pass (<a name="0028-GOVE-189" href="#0028-GOVE-189">0028-GOVE-189</a>)
```

Comment:
```suggestion
- Given a proposal with a closing time less than `governance.proposal.cancellationThreshold` from the current time, if the party that created the proposal votes against it, the proposal will not be instantaneously rejected. At closing time, if the proposal meets the criteria to be accepted it will pass (<a name="0028-GOVE-189" href="#0028-GOVE-189">0028-GOVE-189</a>)
```